src/test/java/seedu/address/logic/commands/ClientEditCommandTest.java

```diff
@@ -75,7 +89,7 @@ public void execute_someFieldsSpecifiedUnfilteredList_success() {
 
         Model expectedModel = new ModelManager(new TbmManager(model.getTbmManager()), new UserPrefs());
         expectedModel.setClient(lastClient, editedClient);
-
+        expectedModel.initialiseTagNoteMap();
```

Comment:
If this is needed to be called everytime we create a modelManager, shouldn't we just put it inside the constructor?